test/gurka/gurka.cc

```diff
@@ -374,8 +374,9 @@ inline void build_pbf(const nodelayout& node_locations,
   objects.sort(object_order_type_unsigned_id_version{});
 
   // Write out the objects in sorted order
-  auto out = osmium::io::make_output_iterator(writer);
-  std::copy(objects.begin(), objects.end(), out);
```

Comment:
this was raising an error on clang. std::copy was missing a conversion between the two iterator types. seems it got stricter with what it allows.